packages/flutter/test/material/list_tile_test.dart

```diff
@@ -111,9 +112,12 @@ void main() {
     }
 
     void testVerticalGeometry(double expectedHeight) {
-      expect(tester.getSize(find.byType(ListTile)), new Size(800.0, expectedHeight));
-      if (hasSubtitle)
+      final Rect tileRect = tester.getRect(find.byType(ListTile));
```

Comment:
We should also verify that the top of the title is > 0. Once the MD team has signed of on a min vertical padding value (or some algorithm) we could refine testVerticalGeometry() further

Reply:
Done.  Also verified that the top centers the title in the widget.